status/views.py

```diff
@@ -0,0 +1,107 @@
+"""
+Status checks for LORE.
+
+Notes:
+
+* Useful messages are logged, but "no config found" is returned whether
+  settings are missing or invalid, to prevent information leakage.
+* Different services provide different information, but all should return
+  "up," "down", or "no config found" for the "status" key.
+"""
+
+import re
+from datetime import datetime
+import logging
+
+from django.conf import settings
+from django.http import JsonResponse
+from elasticsearch import Elasticsearch, ConnectionError as ESConnectionError
+from psycopg2 import connect, OperationalError
+from redis import StrictRedis, ConnectionError as RedisConnectionError
+
+log = logging.getLogger(__name__)
+
+
+def get_pg_info():
+    """Check PostgreSQL connection."""
+    try:
+        conf = settings.DATABASES['default']
+    except (AttributeError, KeyError):
```

Comment:
I actually want these to raise up.  We shouldn't be catching anything in a health check in general, since the status code is what most health checkers use to determine health (the json is just a nice to have, but the status code is the critical thing)